test/lolex-test.js

```diff
@@ -2053,6 +2053,17 @@ describe("lolex", function () {
                 clock.runAll();
                 assert(!called);
             });
+
+            it("passes arguments when installed - GitHub#122", function () {
+                var clock = this.clock = lolex.install();
```

Comment:
why assign to `this.clock`? It is never used in the code. It seems quite sufficient to only set the local variable.